### PR TITLE
feat(DTFS2-5659): extend component test coverage TFM UI

### PR DESCRIPTION
**Pull request by vamshidharukef**
Introduction ✏️
Improve coverage for component tests for TFM UI

Resolution ✔️
Added more component tests for TFM UI

### CODE REVIEW

**Review by lmarrai-sw — DISMISSED** [review]
Thanks for this @vamshidharukef!

**Review by MarRobSoftwire — APPROVED** [review]

**Review by MarRobSoftwire — APPROVED** [review]

File: trade-finance-manager-ui/component-tests/accessibility/accessibility.component-test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const pageRenderer = require('../pageRenderer');
+
+const page = '../templates/accessibility-statement.njk';
+
+const render = pageRenderer(page);
+
+describe(page, () => {
+  let wrapper;
+  const params = {
+    dealId: '1234',
```
Comment (avaitonis) [inline trade-finance-manager-ui/component-tests/accessibility/accessibility.component-test.js:10]: Most likely this id parameter is not required.

Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/accessibility/accessibility.component-test.js]: deaId is removed

**Review by avaitonis — APPROVED** [review]
LGTM

**Pull request merged**